tests/manual/events/KeyboardTest.java

```diff
@@ -38,6 +39,7 @@
 import javafx.scene.control.TextArea;
 import javafx.scene.input.KeyCharacterCombination;
 import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyCombination;
```

Comment:
I wonder if it is possible to add a panel to the test stage with explanations that are currently in the class-level javadoc?  Specifically, the fact that you need to configure a particular layout.

I had some work done to simplify the boilerplate in #1747, though it did not make it yet.